backend/src/main/java/com/kodstar/backend/model/dto/Issue.java

```diff
@@ -1,4 +1,18 @@
 package com.kodstar.backend.model.dto;
 
+import com.kodstar.backend.model.enums.Label;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Set;
+
+@Getter
+@AllArgsConstructor
 public class Issue {
+
+    private Long id;
+    private String tittle;
```

Comment:
suggestion
--> should be written like this "title"

Reply:
will be fixed the typo. Thanks